### DISCUSSION TITLE
Close issue as duplicate.

### DISCUSSION THREAD

**Discussion by RGBCube** [community/community#78159] [Projects and Issues] [answered]
Currently, you can only close as completed/not planned, which confuses people when they see an issue that was close, but not implemented, making them think it was not planned.

A "Close as duplicate of <Enter Number]" option would great, and would mitigate this!

**Comment by erickdc7** [comment]
To close an issue as a duplicate on GitHub, you can add a comment to the issue stating that it's a duplicate and referencing the original issue. For example, you can write "Duplicate of #[issue number]" in the comment, and GitHub will recognize it as a duplicate¹. This method helps to link related issues together and makes it clear to anyone viewing the issue that it has been closed because it is a duplicate, not because it was not planned or completed.

Here's an example of how you can mark an issue as a duplicate:

```markdown
Duplicate of #[issue number]
```

Replace `[issue number]` with the number of the issue that is the original or master issue. Once you've added this comment, you can close the issue manually. This process helps to maintain clarity for other contributors and users who may come across the issue later.

If you have write access to the repository, marking an issue as a duplicate will create a "marked as duplicate" timeline event in the issue's history, which further clarifies the reason for closing the issue. Remember to use this feature responsibly to keep the project's issue tracker organized and informative for all contributors.

**Reply by RGBCube** [reply]
Yes, but it is not visible on the issue list. Making it visible there would be a big win.

**Comment by RiskyMH** [comment, marked as answer]
Yay https://github.blog/changelog/2024-12-12-github-issues-projects-close-issue-as-a-duplicate-rest-api-for-sub-issues-and-more/#close-an-issue-as-a-duplicate

**Reply by roke-julian-lockhart** [reply]
@RiskyMH, it's now at [`changelog/2024-12-12-github-issues-projects-close-issue-as-a-duplicate-rest-api-for-sub-issues-and-more/#broom-close-an-issue-as-a-duplicate`](https://github.blog/changelog/2024-12-12-github-issues-projects-close-issue-as-a-duplicate-rest-api-for-sub-issues-and-more/#broom-close-an-issue-as-a-duplicate:~:text=When%20closing%20an%20issue%2C%20select%20Close%20as%20duplicate%20from%20the%20dropdown%20to%20search%20for%20and%20select%20the%20duplicate%20issue%2E%20You%E2%80%99ll%20then%20see%20an%20event%20in%20the%20timeline%20and%20note%20at%20the%20top%20making%20it%20clear%20why%20it%20was%20closed%2E):

> When closing an issue, select <kbd>Close as duplicate</kbd> from the dropdown to search for and select the duplicate issue. You'll then see an event in the timeline and note at the top making it clear why it was closed.
